src/ChunkData.cpp

```diff
@@ -44,26 +44,6 @@ cChunkData::cChunkData(cAllocationPool<cChunkData::sChunkSection> & a_Pool) :
 
 
 
-cChunkData::~cChunkData()
-{
-	#if __cplusplus < 201103L
-		// auto_ptr style interface for memory management
-		if (!m_IsOwner)
-		{
-			return;
-		}
-	#endif
-	for (size_t i = 0; i < NumSections; i++)
-	{
-		Free(m_Sections[i]);
-		m_Sections[i] = nullptr;
-	}
-}
```

Comment:
Because it was written before the project moved to requiring c++11. I used c++11 move constructors to ensure that I didn't accidentally copy the class, because thats expensive.

Basicly, everything in a __cplusplus < 201103L wrapper is obsolete, and needs to be removed when someone gets round to it. All the __cplusplus >= 201103L wrappers can be removed, because thats guarenteed now.